packages/core/orchestration/src/transaction/transaction-orchestrator.ts

```diff
@@ -798,7 +814,7 @@ export class TransactionOrchestrator extends EventEmitter {
       flow.state = TransactionState.INVOKING
       flow.startedAt = Date.now()
 
-      if (this.options?.store) {
+      if (this.getOptions().store) {
```

Comment:
is store guaranteed here?

```suggestion
      if (this.getOptions()?.store) {
```

Reply:
ignore this. I read it wrong :D